src/Engine/ProtoCore/RuntimeStatus.cs

```diff
@@ -18,7 +18,7 @@ public enum WarningID
             Default,
             AccessViolation,
             AmbiguousMethodDispatch,
-            AurgumentIsNotExpected,
+            ArgumentIsNotExpected,
```

Comment:
@aparajit-pratap I don't think we can change these now, they are public. 😢 maybe just add a todo?